src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -96,6 +96,7 @@ export const RegisterForm: FC = () => {
   })
 
   const onSubmit: SubmitHandler<RegisterFormValues> = (data) => {
+    reset({}, {keepValues: true})
```

Comment:
```suggestion
    reset({keepValues: true})
```

toto by malo stacit, inak fajne :rocket: